projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -365,13 +383,24 @@ export default () => (
         <h3>Gene constraint</h3>
         <ul>
           <li>
-            <ExternalLink href="https://console.cloud.google.com/storage/browser/gnomad-public/release/2.1/ht/constraint">
-              Hail Table (gs://gnomad-public/release/2.1/ht/constraint/constraint.ht)
+            <ExternalLink href="https://console.cloud.google.com/storage/gnomad-public/release/2.1.1/constraint/gnomad.v2.1.1.lof_metrics.by_transcript.ht">
+              Hail Table
+              (gs://gnomad-public/release/2.1.1/constraint/gnomad.v2.1.1.lof_metrics.by_transcript.ht)
+            </ExternalLink>
+          </li>
+          <li>
+            <ExternalLink href="https://storage.googleapis.com/gnomad-public/release/2.1.1/constraint/gnomad.v2.1.1.lof_metrics.by_transcript.txt.bgz">
+              TSV (gnomad.v2.1.1.lof_metrics.by_transcript.txt.bgz)
```

Comment:
Now that there are multiple gene constraint TSVs, I think these should have better labels than "Hail Table" and "TSV".

Maybe
* LoF Metrics by Transcript (Hail Table)
* LoF Metrics by Transcript (TSV)
* LoF Metrics by Gene (TSV)
* LoF Metrics Downsamplings (TSV)

@konradjk

Reply:
Yep, sounds good